bin/kona-exex/src/main.rs

```diff
@@ -1,7 +1,22 @@
+use clap::Parser;
 use eyre::Result;
+use reth::cli::Cli;
+use reth_node_ethereum::EthereumNode;
 
-#[tokio::main]
-async fn main() -> Result<()> {
-    println!("Hello, kona!");
-    Ok(())
+mod kona;
+use kona::KonaExEx;
+
+mod cli;
+use cli::KonaArgsExt;
+
+/// The identifier of the Kona Execution Extension.
+const EXEX_ID: &str = "kona";
+
+fn main() -> Result<()> {
+    Cli::<KonaArgsExt>::parse().run(|builder, args| async move {
+        let node = EthereumNode::default();
```

Comment:
qq: should we try to init the ROLLUP_CONFIG here or is it better to panic! inside KonaExEx::new?

Reply:
probably better to do rollup config init here if it's fallible, unless you expect `new` to also be fallible, since run returns a `Result`

Reply:
for now we don't expect new() to be fallible, ty!

Reply:
I think @Rjected meant the rollup config initialization is fallible - which it should be in your draft.

```rust
        let Some(cfg) = ROLLUP_CONFIGS.get(&args.l2_chain_id).cloned().map(Arc::new) else {
            bail!("Rollup configuration not found for L2 chain id: {}", args.l2_chain_id);
        };
```

Since `ROLLUP_CONFIGS` is a hashmap in the `superchain-registry`, the kv pair may not exist so this becomes fallible right

Reply:
@refcell not sure I follow - if the config doesn't exist for the specified chain id we should exit the exex, correct?